projects/addon-editor/components/toolbar-new/toolbar-new.component.ts

```diff
@@ -272,3 +269,7 @@ export class TuiToolbarNewComponent {
         }
     }
 }
+
+function toolsAssertion(tools: ReadonlyArray<TuiEditorTool>): boolean {
```

Comment:
Are you sure using non-exported functions in decorators does not break AOT?

Reply:
Yes, they will work well.
Example: https://github.com/Tinkoff/taiga-ui/blob/main/projects/kit/components/slider/slider-key-steps.directive.ts#L109